Rakefile

```diff
@@ -9,4 +9,4 @@ task :spec do
   sh "bundle exec bacon #{specs('**')}"
 end
 
-task default: :specs
+task default: :spec
```

Comment:
Yes, there is. A few lines above you declare `task :spec`. ;)

Proof of why this needs fixing, output before this commit:
```
~/Workspace/github/xcode-install(master ✗) rake
rake aborted!
Don't know how to build task 'specs'

Tasks: TOP => default
(See full trace by running task with --trace)
```